products/compute/terraform.yaml

```diff
@@ -1187,6 +1187,27 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       name: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateGCPName'
+  RegionNetworkEndpointGroup: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "region_network_endpoint_group_functions"
+        primary_resource_id: "function_neg"
+        vars:
+          neg_name: "function-neg"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "region_network_endpoint_group_cloudrun"
+        primary_resource_id: "cloudrun_neg"
+        vars:
+          neg_name: "cloudrun-neg"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "region_network_endpoint_group_appengine"
+        primary_resource_id: "appengine_neg"
+        vars:
+          neg_name: "appengine-neg"
+    properties:
```

Comment:
I would also recommend overriding 'region' here like we do for most other resources.